src/app/components/Disclaimer/index.tsx

```diff
@@ -0,0 +1,78 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext, PropsWithChildren } from 'react';
+import useToggle from '#hooks/useToggle';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import { GridItemLarge } from '#components/Grid';
+import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
+import isEmpty from 'ramda/src/isEmpty';
+import { ARTICLE_PAGE } from '../../routes/utils/pageTypes';
+import Paragraph from '../Paragraph';
+import InlineLink from '../InlineLink';
+import styles from './index.styles';
+
+type Props = {
+  increasePaddingOnDesktop?: boolean;
+};
+
+const DisclaimerComponent = ({
+  increasePaddingOnDesktop,
+}: PropsWithChildren<Props>) => {
+  const { disclaimer, translations } = useContext(ServiceContext);
+  const { pageType } = useContext(RequestContext);
+  const { enabled } = useToggle('disclaimer');
+
+  const shouldShow = enabled && disclaimer && !isEmpty(disclaimer);
+
+  if (!shouldShow) return null;
+
+  const infoBannerLabelTranslation = pathOr(
+    'Information',
+    ['infoBannerLabel'],
+    translations,
+  );
+
+  return (
+    <GridItemLarge>
+      <section
+        css={[
+          styles.infoBanner,
+          increasePaddingOnDesktop && styles.increasePaddingOnDesktop,
+        ]}
+        role="region"
+        aria-label={infoBannerLabelTranslation}
+      >
+        <Paragraph
```

Comment:
Could we please change this to a strong element as per the screen reader UX?

Reply:
Sorry I missed this! Will update